### DISCUSSION TITLE
Is it possible to get rid of the warning about not using selectors in the console (at development time) ?

### DISCUSSION THREAD

**Discussion by serge-hulne** [mint-lang/mint#564] [Answered] [answered]
<img width="916" alt="Capture d’écran 2022-12-20 à 16 46 53" src="https://user-images.githubusercontent.com/303502/208707903-80859098-de62-4f42-9b1f-fceeff1b4638.png">

**Comment by gdotdesign** [comment, marked as answer]
Not currently, but if there is a need, we can add a flag to the `start` command to remove it. As a workaround, you can just toggle the `warning` tag in the console.
